docs-source/content/userguide/managing-fmw-domains/soa-suite/Monitor-Exporter-setup.md

```diff
@@ -0,0 +1,286 @@
+Set the context-root to "wls-exporter" for both the deployments. Select "Install and start application" option.
+
+![Set the application name](images/Monitor-Exporter_set-the-applications-name.png) 
+
+Then deploy weblogic-monitor-exporter application and then activate changes for the application to start.  
+If the application started and the port is exposed, then the url "http://<server:port>/wls-exporter" will be accessible.  
+
+![Access weblogic monitor UI](images/Monitor-Exporter_access-weblogic-monitor-ui.png)   
+
+## Prometheus Operator Configuration
+
+Now you have to configure Prometheus to collect the metrics from the weblogic-monitor-exporter. Prometheus operator identifies the targets using service discovery. To get the weblogic-monitor-exporter end point discovered as target, we will need to create a service monitor pointing to the service. Below the sample service monitor deployment yaml.  
+
+### ServiceMonitor for wls-exporter:
+```
+apiVersion: v1
+kind: Secret
+metadata:                 
+  name: basic-auth 
+  namespace: monitoring 
+data:          
+  password: V2VsY29tZTE= # Welcome1                                                                                                                          
+  user: d2VibG9naWM=  # weblogic
+type: Opaque
+---               
+apiVersion: monitoring.coreos.com/v1
+kind: ServiceMonitor
+metadata:            
+  name: wls-exporter-soainfra
+  namespace: monitoring
+  labels: 
+    k8s-app: wls-exporter
+spec:
+  namespaceSelector:
+    matchNames:       
+    - soans             
+  selector:
+    matchLabels:          
+      weblogic.domainName: soainfra
+  endpoints:               
+  - basicAuth:
+      password:    
+        name: basic-auth        
+        key: password
+	  username:                  
+        name: basic-auth       
+        key: user            
+    port: default       
+    relabelings:     
+      - action: labelmap     
+        regex: __meta_kubernetes_service_label_(.+)
+    interval: 10s
+    honorLabels: true
+    path: /wls-exporter/metrics
+```  
+
+The scraping of metrics from wls-exporter requires basic auth. So a secret is created with username/password are base64 encoded. Be careful in the generation of base64 encoded string of user and password. New line character could be included in the encoded string and cause authentication failure. To avoid new line string, use the following sample:  
+```
+$ echo -n "Welcome1" | base64 
+V2VsY29tZTE=
+``` 
+
+In this deployment, it is used "weblogic.domainName: soainfra" as label for selector→matchLabels. So all the server services will be selected for the service monitor. Otherwise you may have to create separate service monitors for the servers if the server name is used as matching labels. The relabelings configuration is required as Prometheus by default ignores the labels provided in the wls-exporter.  
```

Comment:
I'm not sure what is communicated by the arrow between selector and matchLabels.  Could this just be a period if it represents ownership?  Also, I'm not sure what was meant by "relabelings".